#no input request
---
int32 goal_reached
int32 goal_cancelled
